srcpkgs/zenpower3-dkms/template

```diff
@@ -0,0 +1,28 @@
+# Template file for 'zenpower3-dkms'
+pkgname=zenpower3-dkms
+_pkgname=zenpower3
+version=0.2.0
+revision=1
+archs="x86_64"
+wrksrc="${_pkgname}-${version}"
+depends="dkms"
+short_desc="Linux kernel driver for reading sensors for AMD Zen family CPUs (DKMS)"
+maintainer="Anton Afanasyev <anton@doubleasoftware.com>"
+license="GPL-2.0-only"
+homepage="https://github.com/Ta180m/zenpower3"
+distfiles="https://github.com/Ta180m/zenpower3/archive/v${version}.tar.gz"
+checksum=b206c1dfd8ea742a802126d012404d31ae62f2a23ae4723ea1c0f9f54581f6a3
+dkms_modules="${_pkgname} ${version}"
+
+do_install() {
+	vmkdir /usr/src/${_pkgname}-${version}
+	vcopy "./*" usr/src/${_pkgname}-${version}
+	sed -e "s/@CFLGS@//" \
```

Comment:
Using `vsed` is preferred. Could mention that this is taken from the makefile, too.